nixos/modules/home/default.nix

```
@@ -67,9 +67,12 @@
   };
 
   # Enable/Disable the nh module
-  custom.nh = {
-    enable = true;
-    flake = "${builtins.getEnv "HOME"}/toolbox/nixos";
+  custom = {
+    nh = {
+      enable = true;
+      flake = "${builtins.getEnv "HOME"}/toolbox/nixos";
```

May we change this to

```suggestion
      flake = "${builtins.getEnv "HOME"}/Git/toolbox/nixos";
```

and maybe have an environment variable for our like absolute base path to our nixos directory. IE:

~/Git/toolbox/nixos

---

No problem, that's related to nh though not yazi. I'll open another PR with this change as `feat/nh`